corehq/tests/optimizer.py

```diff
@@ -122,6 +119,8 @@ def _extract_tests(test_suite_or_test):
 @nottest
 @contextmanager
 def optimize_apps_for_test_labels(test_labels):
+    # TODO make this work on Django 1.8 (django.test.simple has been removed)
```

Comment:
Upgrading to django 1.8 will break ```optimize_apps_for_test_labels```.  I'm not sure what the alternatives are to ```build_test```.